docxcompose/properties.py

```diff
@@ -1,10 +1,65 @@
 from copy import deepcopy
 from datetime import datetime
+from docx.opc.constants import CONTENT_TYPE as CT
 from docx.opc.constants import RELATIONSHIP_TYPE as RT
+from docx.opc.oxml import serialize_part_xml
+from docx.opc.packuri import PackURI
+from docx.opc.part import Part
 from docx.oxml import parse_xml
 from docx.oxml.coreprops import CT_CoreProperties
+from docxcompose.utils import NS
 from docxcompose.utils import xpath
+from six import string_types
 from six import text_type
+import os.path
+
+
+CUSTOM_PROPERTY_FMTID = '{D5CDD505-2E9C-101B-9397-08002B2CF9AE}'
+CUSTOM_PROPERTY_TYPES = {
+    'text': '<vt:lpwstr xmlns:vt="{}"/>'.format(NS['vt']),
+    'int': '<vt:i4 xmlns:vt="{}"/>'.format(NS['vt']),
+    'bool': '<vt:bool xmlns:vt="{}"/>'.format(NS['vt']),
+    'datetime': '<vt:filetime xmlns:vt="{}"/>'.format(NS['vt']),
+    'float': '<vt:r8 xmlns:vt="{}"/>'.format(NS['vt']),
+}
+
+
+def value2vt(value):
+    if isinstance(value, bool):
+        el = parse_xml(CUSTOM_PROPERTY_TYPES['bool'])
+        el.text = 'true' if value else 'false'
+    elif isinstance(value, int):
+        el = parse_xml(CUSTOM_PROPERTY_TYPES['int'])
+        el.text = text_type(value)
+    elif isinstance(value, float):
+        el = parse_xml(CUSTOM_PROPERTY_TYPES['float'])
+        el.text = text_type(value)
+    elif isinstance(value, datetime):
+        el = parse_xml(CUSTOM_PROPERTY_TYPES['datetime'])
+        el.text = value.strftime('%Y-%m-%dT%H:%M:%SZ')
+    elif isinstance(value, string_types):
+        el = parse_xml(CUSTOM_PROPERTY_TYPES['text'])
+        el.text = value
+    else:
+        raise TypeError('Unsupported type {}'.format(type(value)))
+    return el
+
+
+def vt2value(element):
+    tag = element.tag.split('}')[-1]
```

Comment:
isn't there a more elegant way to get rid of the namespace?

Reply:
At least I'm not aware of one. Suggestions?

Reply:
Yes i think his may work: https://lxml.de/api/lxml.etree.QName-class.html
At the cost of an explicit `lxml` dependency which should be already there implicitly as i'm pretty sure it is required by `python-docx`.

Reply:
Using QName now